test/color/setting_test.dart

```diff
@@ -180,6 +180,63 @@ void main() {
 
         await screenMatchesGolden(tester, 'color_setting_nostroke-example-1');
       });
+
+      processingLegacySpecTest('background in clear() replace background in setup() and drawn rect', (tester) async {
+        await tester.pumpWidget(
+          Processing(
+            sketch: Sketch.simple(
+              draw: (s) {
+                s
+                  ..noLoop()
+                  ..fill(color: const Color(0xFF969696))
+                  ..rect(rect: Rect.fromLTWH(30, 20, 55, 55))
+                  ..clear();
+              },
+            ),
+          ),
+        );
+
+        await screenMatchesGolden(tester, 'color_setting_background_draw_then_clear');
```

Comment:
Notice that other test files include the name of the Processing method under test. This one doesn't. It should begin with `color_setting_clear...`. Please make those adjustments to each of the test files.

Reply:
Yep. I made a mistake here. Will do